### DISCUSSION TITLE
Total number of unwatched episodes

### DISCUSSION THREAD

**Discussion by kamil-zbroniec** [javikr/sofatime#35] [Ideas]
Hey!
I love the app, but I also find it unintuitive to get a number of TV show episodes left at a glance. I always need to remember to add one to the number displayed in the app. I was wondering whether it could be changed or maybe set according to user preference in app settings. 
Thank you for the app though, it's really great.

**Comment by javikr** [comment]
Hi there! Yes, I appreciate your feedback. Please let me investigate how I can change the way we display left episodes number in order to fit with the current UI
